invenio_accounts/templates/semantic-ui/invenio_accounts/register_user.html

```diff
@@ -48,7 +48,7 @@ <h3 class="ui header">
 
     <div class="ui secondary segment padded">
       {{ _('Already have an account?') }}
-      <a href="{{ url_for_security('login') }}">{{ _('Log In') }}</a>
+      <a href="{{ url_for_security('login', next=request.args.get('next')) }}">{{ _('Log In') }}</a>
```

Comment:
Minor: Shouldn't the parameter be validated before being included? Here we're directly passing through a query parameter from a client, hence an attacker should create a link and send it to a victim?

Reply:
Indeed, probably taken from `invenio-oauthclient`, but we actually run the `get_safe_redirect_target` utility there inside the handlers... I think flask-security has some form of protection but not the same as we use

Reply:
After some further discussion with @slint we concluded that there is no improvement by additionally validating the URL through ```get_safe_redirect_target``` since the flask-security form and redirect function will be enforcing its own validator limiting the ```next``` options to subdomains. Currently, it seems that there is more effort required to change this and it doesn't seem that it's in the scope of this PR to do so.